configs/eslint-common-react.json

```diff
@@ -1,10 +1,8 @@
 {
     "extends": [
-        "./eslint-common.json",
         "airbnb",
         "prettier/react",
-        "plugin:prettier/recommended",
-        "./eslint-ts-overrides.json"
+        "./eslint-common.json"
```

Comment:
Both `plugin:prettier/recommended` and `eslint-ts-overrides.json` are part of `eslint-common.json`, so it was redundant here.